services/ui-src/src/components/cards/ObjectiveProgressCardBottomSection.tsx

```diff
@@ -55,107 +55,93 @@ const wereTargetsMetForObjectiveProgress = (
   );
 };
 
-export const EntityStepCardBottomSection = ({
-  stepType,
+export const ObjectiveProgressCardBottomSection = ({
```

Comment:
Find the `ObjectiveProgressCardBottomSection` naming a little strange since nothing else is named `ObjectiveProgressCard`. It makes me wonder where are the other parts of this card?